packages/core/src/events/TapEvent.ts

```diff
@@ -204,12 +204,17 @@ const applyEvent = (target: Global): void => {
 
   /**
    * Listen to `mousedown` events on the target.
-   * Use this to fire tap events, unless one
+   * Use this to fire tapstart events, unless one
    * has already been triggered from a touch event.
    */
   target.addEventListener(
     'mousedown',
     (event) => {
+      // if mouse is not left click
+      if (event.button !== 0) {
+        return;
+      }
+
```

Comment:
We shouldn't be too specific on whether it's the left or the right button, as this can change. You could use the term 'main' instead and declare it above to be reused across both parts. Then we don't need the comments either.

```suggestion
      if (event.button !== MAIN_BUTTON) {
        return;
      }

```